cmd/publishing-bot/github.go

```diff
@@ -53,7 +53,20 @@ func ReportOnIssue(e error, logs, token, org, repo string, issue int) error {
 	}
 
 	// create new newComment
-	body := fmt.Sprintf("The last publishing run failed: %v\n\n```%s```\n", e, tail(logs, 65000))
+	// body := fmt.Sprintf("The last publishing run failed: %v\n\n```%s```\n", e, tail(logs, 65000))
+
+	body := newLogBuilderWithMaxBytes(65000, logs).
+		addHeading(fmt.Sprintf("The last publishing run failed: %v", e)).
+		addHeading("```").
+		trim("\n").
+		split("\n").
+		filter(func(line string) bool {
```

Comment:
does this mean that all non-`+` lines are skipped?

Reply:
@sttts yep. isn't that the behavior we want?

Reply:
the skipped lines could be found in the file logs

Reply:
Instead we want to show up to 50 `+` lines from the bottom, including all non-`+` lines. Above that we only show 50 non-`+` lines.

Reply:
got it